entrypoint.sh

```diff
@@ -7,6 +7,15 @@ platform --version
 sed -i 's/#   StrictHostKeyChecking ask.*/StrictHostKeyChecking accept-new/' /etc/ssh/ssh_config
 FILENAME="${INPUT_DB_DUMP_FILENAME_BASE}-$(date +%F-%T)"
 
-platform db:dump -v --yes --project "$INPUT_PLATFORMSH_PROJECT" --environment "$INPUT_PLATFORMSH_ENVIRONMENT" --gzip -f "$FILENAME".sql.gz
+# Check if the optional relationship value exists.
+RELATIONSHIP=""
+if [ -z "${INPUT_PLATFORMSH_RELATIONSHIP}" ]
+then
+  echo "No relationship variable provided so no relationship parameter added to platform db:dump."
+else
+  RELATIONSHIP="--relationship $INPUT_PLATFORMSH_RELATIONSHIP"
```

Comment:
@mmatsoo I guess there isn't a "default" value for `--relationship VALUE` that we could always specify if the argument is not set? Platform.sh just "figures it out" if you don't specify one?

Reply:
@markdorison Yep, that seems to be the case since you don't need the `--relationship` parameter when there's only one DB service in use.